src/api/handlers/member.ts

```diff
@@ -124,7 +124,7 @@ export async function sendDirectMessage(
   if (!dmChannel) {
     // If not available in cache create a new one.
     const dmChannelData = await RequestManager.post(
-      endpoints.USER_CREATE_DM,
+      endpoints.USER_DM,
```

Comment:
@itohatweb can you revert this back?

Reply:
I have changed these endpoints in handlers already